src/sass/_mixins.scss

```diff
@@ -107,3 +107,7 @@
   position: static;
   width: auto;
 }
+
+@mixin z-index($value) {
```

Comment:
This mixin ensures that z-index is a multiple of 10.

Reply:
Nice solution to the enforced multiples of 10.

Reply:
I think this is over-engineering as the amount of times we use z-index is only going to be a small amount.

Reply:
Maybe I can adapt the mixin to throw a warning if we use it too much. :). I will let it in for now and we can remove/refactor if it proves to be annoying in practice.

Thanks for the review.

Reply:
It's not annoying there is just no need for it. I put z-index in the same pot as position absolute where you only use it when you only need to. Very rarely.